analyzer.go

```diff
@@ -86,14 +86,28 @@ func (a *Analyzer) skipCodeGen(file *ast.File) ([]*ast.CommentGroup, []*ast.Comm
 	if len(comments) > 0 {
 		list = comments[0].List
 	}
-	if len(comments) > 0 && strings.Contains(comments[0].Text(), "DO NOT EDIT") {
+	if len(comments) > 0 && (strings.Contains(comments[0].Text(), "DO NOT EDIT") || strings.Contains(comments[0].Text(), "go:build")) {
```

Comment:
You are only ignoring one of the go directives.
This is a very naive implementation.

Reply:
Fixed.

Reply:
not fixed. https://github.com/denis-tingaikin/go-header/pull/51#discussion_r2094610321

Reply:
added commit https://github.com/denis-tingaikin/go-header/pull/51/commits/c3631e75f81e202d4bc3a5834f7d9fe39fd32826